pebbles/views/sessions.py

```diff
@@ -28,12 +30,22 @@ def post(self):
 
         user = User.query.filter_by(email=form.email.data).first()
         if user and user.check_password(form.password.data):
+            if user.is_admin:
+                icons = json.dumps(["Dashboard", "Users", "Groups", "Blueprints", "Configure", "Statistics", "Account"])
```

Comment:
The list of tabs in future may increase, consider putting these values in a separate data structure or file, from where it would be easy to change directly.

Reply:
It is in just one place. And it makes more sense to be in the same file. But I just put it in the top for more clarity.